tests/cephfs/snapshot_clone/cephfs_cg_io.py

```diff
@@ -0,0 +1,676 @@
+"""
+This is cephfs consistency group snapshot feature IO module
+It contains methods to start IO on quiesce-set/consistency group
+
+"""
+import datetime
+import random
+import string
+import time
+from multiprocessing import Value
+from threading import Thread
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from tests.cephfs.snapshot_clone.cg_snap_utils import CG_Snap_Utils as CG_snap_util
+from utility.log import Log
+
+log = Log(__name__)
+cg_io_status = {}
+
+
+class CG_snap_IO(object):
+    def __init__(self, ceph_cluster):
+        """
+        FS Snapshot Utility object
+        Args:
+            ceph_cluster (ceph.ceph.Ceph): ceph cluster
+        """
+
+        self.ceph_cluster = ceph_cluster
+        self.mons = ceph_cluster.get_ceph_objects("mon")
+        self.mgrs = ceph_cluster.get_ceph_objects("mgr")
+        self.osds = ceph_cluster.get_ceph_objects("osd")
+        self.mdss = ceph_cluster.get_ceph_objects("mds")
+        self.clients = ceph_cluster.get_ceph_objects("client")
+        self.cg_snap_util = CG_snap_util(ceph_cluster)
+        self.fs_util = FsUtils(ceph_cluster)
+
+    def start_cg_io(
+        self, clients, qs_members, client_mnt_dict, cg_test_io_status, io_run_time
+    ):
+        """
+        Start IO quiesce-set members and validate IO progress based on quiesce status
+        Args:
+            client: ceph client to mount subvolumes and run IO
+            qs_id : A list of quiesce members, each member in format subvol1 or subvol1/group1 if
```

Comment:
variable used in function definition and documentation is different. please add documentation for all the args of the function. The format for subvolumes of non-default group must be "group/subvolume" where group comes first and then subvolume.

Reply:
As mentioned in description, please ignore doc section for first review, its not up-to-date, i will push the doc content by tomorrow.

Reply:
doc strings added to cg_snap_utils.py, cephfs_cg_io.py